packages/apps/src/SideBar/index.tsx

```diff
@@ -33,45 +30,24 @@ class SideBar extends React.PureComponent<Props> {
 
     return (
       <div className='apps--SideBar'>
-        <Menu
-          secondary
-          vertical
-        >
-          <img
-            alt='polkadot'
-            className='apps--SideBar-logo'
-            src={LOGO}
-          />
-          {
-            routing.routes
-              .filter((route) =>
-                !route || !route.isHidden
-              )
-              .map((route, index) => (
-                route
-                  ? (
-                    <Item
-                      key={route.name}
-                      t={t}
-                      route={route}
-                    />
-                  )
-                  : (
-                    <Menu.Divider
-                      hidden
-                      key={index}
-                    />
-                  )
-              ))
-          }
+        <Menu secondary vertical>
+          <img alt='polkadot' className='apps--SideBar-logo' src={LOGO} />
+          {routing.routes
+            .filter(route => !route || !route.isHidden)
+            .map((route, index) =>
+              route ? <Item key={route.name} t={t} route={route} /> : <Menu.Divider hidden key={index} />
```

Comment:
Prefer 

```
route
  ? <Item key={route.name} t={t} route={route} />
  : <Menu.Divider hidden key={index} />
```

(as per the original - whitespace doesn't cost anything, but aids readability)